adapters/handlers/grpc/v1/parse_search_request.go

```diff
@@ -330,12 +330,16 @@ func (p *Parser) Search(req *pb.SearchRequest, config *config.Config) (dto.GetPa
 	}
 
 	if req.Filters != nil {
-		clause, err := ExtractFilters(req.Filters, p.getClass, req.Collection)
+		// Todo: Replace with real auth function
```

Comment:
Will this TODO be handled in another PR?

Reply:
yes, this requires more changes at other places and I wanted to minimize the amount of cahnges in one PR :)